homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/about.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>About</title>
+        <link href="./styles/style.css" rel="stylesheet" type="text/css">
+    </head>
+    <body>
+        <header class="header">
+            <div class="header__flexbox">
+                <img src="./images/logo.svg" class="logo" alt="logo">
+                <nav>
+                    <ul class="nav__items">
```

Comment:
Hey, @Moo0oony, if you like to use BEM naming, you should start from the `block`. Please read through my comments [here](https://github.com/MastersAcademy/fe-course-basic-2023/pull/136/files/6768f073407d5f5a55b8182c191dfbae9e80990a#r1367945552)

Reply:
Hey @vladyslav-yermolin-moc ! I am still determining what I would better use. Is it convenient for now to change all the "almost-BEM-like" names to the names without a specific style but more consistent, I.e. "header-flexbox", "nav-items", etc.? 
Concerning other comments, I am going to edit everything, just decided to inform you in this comment to avoid spamming :)

Reply:
Ok, then U have to decide, which system of CSS class naming will you prefer. It could be:
- kebab-case: `.class`, `.another-class`
- BEM: `.block` > `.block__element`
- camelCase: `oneClass`, anotherClass`
- snake_case: `one_class`, `another_class`

But this system should exist. 
So, if you like, I can insist that you are using BEM and you will learn it in the process. Or I can skip and make your life easier

Reply:
Baseline: all your classes should be made according to one style

Reply:
Thanks for the answer! I guess I'll stick up with the snake_case so I won't mess with the camelCase in JS (thanks for the style names btw). And yeah, for the single class, I understood.